src/main/resources/schema/provider.definition.schema.v1.json

```diff
@@ -207,6 +207,33 @@
                     "additionalProperties": false
                 }
             ]
+        },
+        "serviceLink": {
+            "type": "object",
+            "properties": {
+                "$comment": {
+                    "$ref": "https://json-schema.org/draft-07/schema#/properties/$comment"
+                },
+                "templateUri": {
+                    "type": "string",
+                    "pattern": "^(/|https:)"
+                },
+                "mappings": {
+                    "type": "object",
+                    "patternProperties": {
+                        "^[A-Za-z0-9]{1,64}$": {
+                            "type": "string",
+                            "format": "json-pointer"
+                        }
+                    },
+                    "additionalProperties": false
+                }
+            },
+            "required": [
+                "templateUri",
+                "mappings"
```

Comment:
`mappings` could be optional, right?

Reply:
i don't see how that could produce sensible links to the resource, since you always have to map some kind of information into to the link. and to keep the templating operation simple, you have to map information to variables (you can't put a JSON pointer in the template URI). i guess you could define an empty `mappings`, but this way there's one less check required for consuming code to see if `mappings` is even present.